svelte-kit-scss/.storybook/main.cjs

```diff
@@ -1,25 +1,20 @@
-const sveltePreprocess = require('svelte-preprocess');
 const { mergeConfig } = require('vite');
 const path = require('path');
 
 module.exports = {
-  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx|svelte)'],
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
   addons: [
     '@storybook/addon-links',
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
-    '@storybook/addon-svelte-csf',
-    '@storybook/preset-scss',
+    // '@storybook/preset-scss',
```

Comment:
Is there a specific reason we have this commented out? If not, could we remove it to eliminate dead code?

Reply:
@iansamz also fix failing CI